WLC/image_processing/image_preprocessor.py

```diff
@@ -0,0 +1,15 @@
+import cv2
+
+
+class ImagePreprocessor:
+    def __init__(self):
```

Comment:
Not a python expert but this doesn't seem to do anything, so remove it?

Reply:
Paul added this to all other classes so will keep it just for consistency.